src/commands/reset.php

```diff
@@ -38,6 +38,13 @@
 
 quietly_tear_down_stack();
 
+$lastruntime = root( '/.lastruntime' );
+echo "Removing {$lastruntime} ... ";
+echo ( ! file_exists( $lastruntime ) || unlink( $lastruntime ) ) ?
+	light_cyan( 'done' )
+	: magenta( 'fail, remove it manually' );
```

Comment:
This message, and the one above it, will be printed on a new line in respect to the `Removing .lastruntime ...` line.
For this, they should be "independent" in providing information. 
E.g. `File .lastruntime removed` and `Failed to remove file .lastruntime; you can remove it manually with rm .lastruntime`.
I suggest adding the copy-and-paste remove command for usability purposes.